app/controllers/v2/content_items_controller.rb

```diff
@@ -41,5 +55,11 @@ def publishing_app
         current_user.app_name
       end
     end
+
+    def return_400_if_no_format
```

Comment:
Using `params.fetch` like in the `index` method will raise a `ActionController::ParameterMissing` if the parameter is not provided. http://api.rubyonrails.org/classes/ActionController/Parameters.html#method-i-fetch.